tests/domain/aggregate.unit.coffee

```diff
@@ -68,11 +68,11 @@ describe "Space.eventSourcing.Aggregate", ->
       event = type: 'Test', aggregateId: 'bla'
       expect(=> @aggregate.record(event)).to.throw Aggregate.ERRORS.domainEventRequired
 
-    it 'throws if no handler is defined for the event', ->
+    it 'is ok if no handler is defined for the event', ->
       event = new Event sourceId: '123'
       aggregate = new Aggregate '123'
-      expectedError = Aggregate.ERRORS.cannotHandleMessage + event.typeName()
-      expect(-> aggregate.record event).to.throw expectedError
+      error = Aggregate.ERRORS.cannotHandleMessage + event.typeName()
+      expect(-> aggregate.record event).not.to.throw error
 
     it 'does not append the event to the queue if something fails', ->
```

Comment:
Is it exclusively staging the event for committing? If so maybe this could read `does not stage the event for committing if something fails`. I understand this is outside the scope of this PR, so we can move this discussion somewhere else?